skimage/transform/_warps.py

```diff
@@ -6,6 +6,14 @@
                          _convert_warp_input, _clip_warp_output)
 
 
+def _multichannel_default(multichannel, ndim):
+    # utility for maintaining previous color image default behavior
+    if ndim == 3:
+        return True
+    else:
+        return False
```

Comment:
@jni:  In an earlier commit I had it nearly identical to what you propose above.  For some reason I later changed it so that `_multichannel_default` is only ever called when `multichannel` is None.  

I can put it back as you propose for clarity and add the deprecation warning